packages/eslint-plugin-template/tests/rules/interactive-supports-focus/cases.ts

```diff
@@ -58,8 +58,8 @@ export const valid = [
       <button (click)="onClick()" class="foo">Foo</button>
       <option (click)="onClick()" class="foo">Food</option>
       <select (click)="onClick()" class="foo"></select>
-      <summary (click)="onClick()">Foo</summary>
-      <textarea (keypress)="onKeypress()" class="foo"></textarea>
+      <summary (click)="onClick()" aria-expanded>Foo</summary>
+      <textarea (keypress)="onKeypress()" class="foo" aria-multiline></textarea>
```

Comment:
I think the `role-has-required-aria` rule is a better place for these test cases and resulting docs. The required aria attributes checked by the `role-has-required-aria` rule are generally for communicating state to screen readers. This `interactive-supports-focus` rule checks whether an element can receive focus, which both `<summary>` and `<textarea>` have built-in browser support for.

Reply:
I'm not getting the failure for this case locally but want to debug and figure out why it failed in CI. My initial guess is that `<summary>` is expected to be inside a `<details>` parent.